src/Transport/Topology/Topologies/EndpointOrientedTopologySectionManager.cs

```diff
@@ -193,15 +193,30 @@ TopologySectionInternal BuildSubscriptionHierarchy(Type eventType)
 
             var topics = new List<EntityInfoInternal>();
             var subs = new List<SubscriptionInfoInternal>();
-            foreach (var topicPath in topicPaths)
+            foreach (var publisher in publishers)
             {
+                var topicPath = publisher + ".events";
                 var path = addressingLogic.Apply(topicPath, EntityType.Topic).Name;
-                topics.AddRange(namespaces.Select(ns => new EntityInfoInternal
+
+                var destinationsOutsideTopology = namespaceConfigurations.Where(c => c.RegisteredEndpoints.Contains(publisher)).ToList();
```

Comment:
I used `RegisteredEndpoints.Contains(d, StringComparer.OrdinalIgnoreCase)`

Reply:
makes sense, I'll change it